uftrace.c

```diff
@@ -1006,6 +1006,7 @@ static void free_opts(struct opts *opts)
 	free(opts->retval);
 	free(opts->tid);
 	free(opts->event);
+        free(opts->patch);
```

Comment:
Thanks! It seems that `opts->caller` and `opts->watch` are also missed.

Please also change whitespace to tab.

Reply:
Nice to see you again, @honggyukim 
I reflected your request.

Reply:
Thanks for the update. I think @namhyung will take care of this soon.